src/components/Button.jsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { View } from 'react-native-web';
+import { Button as Btn } from 'react-native-web';
+
+export class Button extends React.Component {
```

Comment:
Is this definition for both react and react-native?

Reply:
It's `react-native-web` that means to use React Native APIs for both Web and Native.
If you use plain React for your components, you can't have React Native portability